app/tests/pages/other-benefits.test.tsx

```diff
@@ -1,24 +1,87 @@
-import { render, screen, waitFor } from '@testing-library/react'
-import { axe } from 'jest-axe'
+import { render, screen } from '@testing-library/react'
+import singletonRouter from 'next/router'
 
 import OtherBenefits from '@pages/other-benefits'
 
-describe('OtherBenefits', () => {
-  it('should render the heading', () => {
-    render(<OtherBenefits />)
+import * as sessionModule from '@src/hooks/useSessionStorage'
+import { initialSessionData } from '@utils/sessionData'
 
-    const heading = screen.getByText(/Your eligibility/i)
+import { setMockSession, setup } from '../helpers/setup'
+import {
+  testAccessibility,
+  testBackLink,
+  testSnapshot,
+} from '../helpers/sharedTests'
 
-    expect(heading).toBeInTheDocument()
-    expect(heading).toMatchSnapshot()
-  })
+/**
+ * Test setup
+ */
 
-  it('should pass accessibility scan', async () => {
-    const { container } = render(<OtherBenefits />)
-    const results = await axe(container)
+const route = '/other-benefits'
+const backRoute = '/eligibility'
+const forwardRoute = '/'
 
-    await waitFor(() => {
-      expect(results).toHaveNoViolations()
-    })
-  })
+/**
```

Comment:
Super random but do we need the "Begin tests" string in all/many of the test files?

Reply:
😅 No I suppose not. Good feedback. Thank you!